internal/resources/acceptance_tests/audit_log_export_acceptance_test.go

```diff
@@ -0,0 +1,222 @@
+					resource.TestCheckResourceAttr(auditLogExportResourceReference, "status", ExportStatusNoAuditLogs),
+				),
+			},
+		},
+	})
+}
+
+// create cluster and enable audit logs
+func testAccAuditLogExportSetup(providerAndVariables, projectName, projectResourceReference, clusterResourceName, clusterResourceReference, cidr, auditLogSettingsResourceName string) string {
+	config := fmt.Sprintf(`
+%[1]s
+
+resource "couchbase-capella_project" "%[2]s" {
+    organization_id = var.organization_id
+	name            = "%[2]s"
+	description     = "terraform audit log export test cluster"
+}
+
+resource "couchbase-capella_cluster" "%[4]s" {
+  organization_id = var.organization_id
+  project_id      = %[3]s.id
+  name            = "terraform audit log export test cluster"
+  description     = "terraform audit log export test cluster"
+  configuration_type = "multiNode"
+  cloud_provider = {
+    type   = "aws"
+    region = "us-east-1"
+    cidr   = "%[6]s"
+  }
+  service_groups = [
+    {
+      node = {
+        compute = {
+          cpu = 4
+          ram = 16
+        }
+        disk = {
+          storage = 50
+          type    = "gp3"
+          iops    = 3000
+        }
+      }
+      num_of_nodes = 3
+      services     = ["data"]
+    }
+  ]
+  availability = {
+    "type" : "multi"
+  }
+  support = {
+    plan     = "enterprise"
+    timezone = "PT"
+  }
+}
+
+resource "couchbase-capella_audit_log_settings" "%[7]s" {
+  organization_id   = var.organization_id
+  project_id        = %[3]s.id
+  cluster_id        = %[5]s.id
+  audit_enabled     = true
+  enabled_event_ids = [8243, 8257, 8265]
+  disabled_users    = []
+}
+
+`, providerAndVariables, projectName, projectResourceReference, clusterResourceName, clusterResourceReference, cidr, auditLogSettingsResourceName)
+
+	return config
+}
+
+func testAccAuditLogExportGetCluster(resourceReference string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		// retrieve the resource by name from state
+
+		var rawState map[string]string
+		for _, m := range s.Modules {
+			if len(m.Resources) > 0 {
+				if v, ok := m.Resources[resourceReference]; ok {
+					rawState = v.Primary.Attributes
+				}
+			}
+		}
+		fmt.Printf("raw state %s", rawState)
+		data, err := acctest.TestClient()
+		if err != nil {
+			return err
+		}
+		err = getCluster(data, rawState["organization_id"], rawState["project_id"], rawState["id"])
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+}
+
+func getCluster(data *providerschema.Data, organizationId, projectId, clusterId string) error {
+	url := fmt.Sprintf("%s/v4/organizations/%s/projects/%s/clusters/%s", data.HostURL, organizationId, projectId, clusterId)
+	cfg := api.EndpointCfg{Url: url, Method: http.MethodGet, SuccessStatus: http.StatusOK}
+	_, err := data.Client.ExecuteWithRetry(
+		context.Background(),
+		cfg,
+		nil,
+		data.Token,
+		nil,
+	)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
This getCluster call is used across many acceptance tests. It would be beneficial to move this to a common file accessible to all. However, if the change is urgent, we can move this later as well.